crates/lib3h/src/engine/ghost_engine_wrapper.rs

```diff
@@ -150,6 +155,14 @@ where
 
         Ok((responses.len() > 0, responses))
     }
+
+    pub fn advertise(&self) -> Url {
+        Url::parse("mem://fixme").unwrap()
```

Comment:
Can't we do `self.engine.as_ref().as_ref().advertise()` ?

Reply:
turns out not!!